source/3-landing-page-cicd/cdk/test/landing-page.test.ts

```diff
@@ -1,13 +1,32 @@
-import { expect as expectCDK, matchTemplate, MatchStyle } from '@aws-cdk/assert';
+import { expect as expectCDK, haveResource } from '@aws-cdk/assert';
 import * as cdk from '@aws-cdk/core';
 import * as LandingPage from '../lib/landing-page-stack';
 
-test('Empty Stack', () => {
-    const app = new cdk.App();
-    // WHEN
-    const stack = new LandingPage.LandingPageStack(app, 'MyTestStack');
-    // THEN
-    expectCDK(stack).to(matchTemplate({
-      "Resources": {}
-    }, MatchStyle.EXACT))
+test('Stack is Tagged', () => {
+  const app = new cdk.App({
+    context: {
+      service_name: 'Landing-Page-CICD'
+    }
+  });
+  const stack = new LandingPage.LandingPageStack(app, 'MyTestStack');
+  expectCDK(stack).to(
+    haveResource('AWS::S3::Bucket', {
+      Tags: [
+        {
+          Key: 'StackType',
```

Comment:
Test can't pass since now it's ServiceName ...

Reply:
oops, sorry :) will fix it now